### ISSUE TITLE
[Bug] [Android] Button Border clipped

### ISSUE THREAD

**Issue by plamen-i** [xamarin/Xamarin.Forms#9235]
Buttons defined as follows:

                <Button Grid.Row="1"
                        Grid.Column="1"
                        Command="..."
                        CommandParameter="..."
                        BackgroundColor="..."
                        BorderColor="#009DFF"
                        BorderWidth="1"
                        CornerRadius="2"
                        FontSize="13"
                        Text="PIN-code"
                        TextColor="White" />

                <Button Grid.Row="1"
                        Grid.Column="2"
                        Command="..."
                        CommandParameter="..."
                        BackgroundColor="..."
                        BorderColor="#009DFF"
                        BorderWidth="1"
                        CornerRadius="2"
                        FontSize="13"
                        Text="Terminal ID"
                        TextColor="White" />

                <Button Grid.Row="1"
                        Grid.Column="3"
                        Command="..."
                        CommandParameter="..."
                        BackgroundColor="..."
                        BorderColor="#009DFF"
                        BorderWidth="1"
                        CornerRadius="2"
                        FontSize="13"
                        Text="Log In Pass"
                        TextColor="White" />

- Version with issue: XF 4.4.0.991477 and XF 4.5.0.142-pre1
- Last known good version: XF 4.4.0.991265
- IDE:  Visual Studio 2019 version 16.4.3
- Platform Target Frameworks:
  - Android: 10.0
- Android Support Library Version: 28.0.0.3
- Affected Devices: Android 7.1 Emulator (1024x768, 120 dpi)

### Screenshots

OK:
![XF v4 4 0 991265 - OK](https://user-images.githubusercontent.com/2354584/72545353-91750480-3891-11ea-9a4c-043be086593a.png)

Border Clipped:
![XF v4 4 0 991477   XF v4 5 0 142-pre1 - Border Clipped](https://user-images.githubusercontent.com/2354584/72545351-91750480-3891-11ea-886f-aaa9c0201ed7.png)

### Workaround

Forms.SetFlags("UseLegacyRenderers") corrects the problem.

**Comment by StephaneDelcroix** [comment]
could you please attach a repro project ? thanks

**Comment by plamen-i** [comment]
Here's repro:

[Repro9235.zip](https://github.com/xamarin/Xamarin.Forms/files/4077975/Repro9235.zip)

**Comment by samhouts** [comment]
@plamen-i Yep, definitely repros on a 120dpi emulator. Do you actually need to support such a device?

**Comment by plamen-i** [comment]
This is taken from app I write currently for 1024/768 10' device by customer requirement. I have no such device and I simulated it with 120 dpi emulator so software looks like on device. Not sure about real dpi on device though, but anyway - something between 120 and 160 dpi.

**Comment by plamen-i** [comment]
More precisely: Samsung Galaxy Tab A 9.7 - 1024x768 - 9.7' - 132 dpi

**Comment by plamen-i** [comment]
Sorry - incidentally closed.

**Comment by plamen-i** [comment]
Any news on this? I'm stuck on XF 4.4.0.991265...
